unpackaged/post/case_behavior_record_types/standardValueSetTranslations/Case%%%NAMESPACE%%%Category__c-es_MX.standardValueSetTranslation

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is a duplicate of the values for the Category__c field on the Case Object Translations file [here](https://github.com/SalesforceFoundation/EDA/blob/d2f811c912f766203c2a32e90c2d32cdd2d7c5a6/src/objectTranslations/Case-es_MX.objectTranslation#L7). Since this field is managed, the location of this translations file is likely inaccurate and should be in the src folder. Please confirm this is the desired state and whether we want to have duplicated translations.